INSTALL.md

```diff
@@ -8,6 +9,12 @@
   * [Circle CI](#circle-ci)
   * [Cirrus CI](#cirrus-ci)
 
+## Homebrew
+
+```bash
+brew install cirruslabs/cli/cirrus
```

Comment:
Is it possible to make this `cirruslabs/cirrus-cli` to match the current repository's owner and name for easier recall (or even `cirrus` or `cirrus-cli`)?

Reply:
There are some requirements in order to be added to the main home brew packages: https://docs.brew.sh/Acceptable-Formulae

For now it seems we need to stick with a tap until CLI is stable at least 😪

Reply:
But regarding the initial question, are there any requirements for the actual naming schema, though?

Reply:
Yeah, `brew install cirruslabs/cli/cirrus` is a short version of

```
brew tap cirruslabs/cli
brew install cirrus
```

Naming convention is that `<organization>/<tap>` tap is located in `github.com/<organization>/homebrew-<tap>` repository so there is not that much flexibility.